app/presenters/facet/collection_presenter.rb

```diff
@@ -2,26 +2,16 @@
 
 module Facet
   class CollectionPresenter < SimplePresenter
-    def add_facet_params(item)
+    def add_facet_url(item)
       value = facet_value_for_facet_item(item)
-
-      if default_facet_value?(value)
-        params.except(:id).merge(controller: :portal, action: :index)
-      else
-        params.merge(controller: :collections, action: :show, id: value)
-      end
+      base_url = default_facet_value?(value) ? search_url : collection_url(value)
+      [base_url, facet_item_url_base_query].join('?')
     end
 
-    ##
-    # Removing the collection facet only works when in a collection,
-    # as it redirects to the standard search.
-    # The only reason it takes a facet_item as a param is because
-    # this overrides {FacetPresenter}
-    #
-    # @param see {#facet_item}
-    # @return [Hash] Request parameters without the collection
-    def remove_facet_params(_item)
-      params.except(:id).merge(controller: :portal, action: :index)
+    # TODO: is this useful behaviour when the collection facet is presented
+    #       like a radio button where one is always selected?
```

Comment:
IMO yes it's useful, because it then behaves like the other facets. Where clicking on a selected facet will deselect that facet.

the 'ordered' method down below, however is not being used as far as I can tell.